blueprint/Blueprint.go

```diff
@@ -0,0 +1,45 @@
+package blueprint
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/beevik/etree"
+)
+
+// Blueprint structure
+type Blueprint struct {
+	XMLData *etree.Document
+}
+
+// CreateBlueprint creates empty Blueprint
+func CreateBlueprint() *Blueprint {
+	return &Blueprint{XMLData: etree.NewDocument()}
+}
+
+// Render method to render blueprint values to string
+func (bp Blueprint) Render() (string, error) {
```

Comment:
Method `CreateBlueprint` returns a pointer to `Blueprint` structure but this method is defined on the structure `Blueprint` itself and not on the pointer. Wouldn't that be a bit inpractical to use?